src/Form/AkamaiClearUrlForm.php

```diff
@@ -23,10 +23,9 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $config = $this->config('akamai.settings');
     $form = array();
 
-    $current_uri = $this->requestStack->getCurrentRequest()->getRequestUri();
+    $current_uri = $this->getRequest()->getRequestUri();
```

Comment:
This was just changed - see https://github.com/d8-contrib-modules/akamai/commit/929cd61ba0a52f92b01975d0df511b4d4dae17ba

Is it better to use requestStack or getRequest()?